07_cmd_lab/cmdlab.txt

```diff
@@ -1,15 +1,43 @@
 Intro To Command Line Lab.
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
-1. Login to your UNIX account. How do you find:
-  - your current shell?
-  - your default shell?
-  - a list of all shells currently installed on the machine?
+1. Login to your UNIX account. 
+How do you find:
+ 
+ 
+- your current shell?
+ 
+dash
+- your default shell?
+  
+bash
+- a list of all shells currently installed on the machine?
 
-עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
+עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא 
+/bin/sh
+/bin/dash
+/bin/bash
+/bin/rbash
+bash
 
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd".
+ Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
-3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+it's a password file. holds per each user one line and 7 fields :
+login name 
+optional encrypted password
+numerical user ID
+user name or comment field
+user home directory
+optional user interpreter 
+
+
+3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+
+
+ls -l --block-size /etc/
+

```

Comment:
Did it work? I tried and got the error:

```
 invalid --block-size argument '/etc'
```

block size controls how ls prints the size column, for example it lets you print the size in bytes, kilo-bytes, MB etc. 

your task is to find another option that sorts according to the file size, but also as a bonus you can try to get block-size to work